kotlin-generator/src/helpers.ts

```diff
@@ -47,13 +49,17 @@ export function getAnnotation(type: Type): string {
 }
 
 export function generateClass(type: StructType) {
-    return `data class ${type.name}(\n${type.fields
+    let classDesc = "@Parcelize\n";
+    classDesc += `    data class ${type.name}(\n${type.fields
         .map(field => {
             let fieldDesc = getAnnotation(field.type);
-            fieldDesc += `        val ${mangle(field.name)}: ${generateKotlinTypeName(field.type)}`;
+            fieldDesc += `        var ${mangle(field.name)}: ${generateKotlinTypeName(field.type)} ${
+                field.type.constructor === OptionalType ? "= null" : ""
+            }`;
```

Comment:
This will leave a " " character after the field if it is not nulable. Add the space before the equal only if the type is optional.